packages/web-components/src/components/va-alert/va-alert.scss

```diff
@@ -109,29 +109,29 @@ i.fa-times-circle::before {
 
 // Override USWDS background and border colors for VA styles.
 .usa-alert--warning {
-  border-color: var(--color-gold);
+  border-color: var(--uswds-system-color-gold-vivid-20);
```

Comment:
If you remove these USWDS CSS overrides completely from the stylesheet, does the USWDS system just naturally display the correct color?

Reply:
For example, the override here seems like it's just applying the same color from default styles of `.usa-alert--error`:

![Screenshot 2023-12-21 at 9 52 01 AM](https://github.com/department-of-veterans-affairs/component-library/assets/872479/8114227a-120c-4c26-b5a0-4bba85a30efe)

Reply:
Yes, it naturally displayed correct colors.

Reply:
Those are coming from css-library, that is how it should be correct?

Reply:
We are importing this module below from the USWDS system and that's applying USWDS styles to `.usa-alert--error` `.usa-alert` etc.

https://github.com/department-of-veterans-affairs/component-library/blob/840b4073909428ab66d4394f1d0afc0ce8162295/packages/web-components/src/components/va-alert/va-alert.scss#L3

It applies the USWDS colors by default. So I'm wondering if we can just delete our VA override CSS completely and let the module do its thing now? I'm not sure that we need to involve the CSS Library tokens at all. 🤷🏻‍♀️

Reply:
Absolutely, I agree with your assessment. I've just tried using the module from the USWDS system as you mentioned, and it works perfectly. This confirms that we can indeed remove our VA override CSS and allow the module to apply the USWDS styles and colors by default. It seems that involving the CSS Library tokens might be unnecessary in this context. I'll go ahead and make that change now.

Thanks for pointing this out! 🙌🏼